packages/microwave/.browserslistrc

```diff
@@ -0,0 +1,16 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
There's browserlistrc file for libs? I thought it was only for apps

Reply:
Actually, this was generated by the Nx cli. I think, that right now it is mainly used for CSS autoprefixing so it might be useful even for libraries that have CSS. Though, Angular build or other tools might use the information from browserlist for other purpose. 

If for some reason, Microwave starts displaying errors in debug mode or something, we might want to produce CSS.
What do you think?